resources/property.py

```diff
@@ -57,6 +56,27 @@ def post(self, id):
 
         return property.json(), 201
 
+class ArchiveProperties(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('ids', action='append')
+
+    @admin_required
+    def post(self):
```

Comment:
I agree, that would be more restful. In the spirit of PATCH being a set of instructions for change, I'm making it a PATCH. In this case it is an idempotent application of PATCH because no matter how many times you hit this archive route, it will end up with the listed properties having their archived field set to True.